golanci-lint/main.go

```diff
@@ -7,4 +7,6 @@ import (
 func main() {
 	s := []string{}
 	fmt.Println("Hello, playground")
+
+	s := []string{}
```

Comment:
🚫 **[golangci-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>no new variables on left side of := (typecheck)